#iZ and CWS
o http://www.openoffice.org/issues/show_bug.cgi?id=37731
o http://eis.services.openoffice.org/EIS2/servlet/cws.ShowCWS?Id=2030&Path=SRC680%2Fpj16
o pj16

Description
Do not export LINK to environment

Index: config_office/set_soenv.in
===================================================================
RCS file: /cvs/tools/config_office/set_soenv.in,v
retrieving revision 1.31
diff -u -r1.31 set_soenv.in
--- config_office/set_soenv.in	5 Jan 2005 12:10:00 -0000	1.31
+++ config_office/set_soenv.in	9 Jan 2005 18:51:24 -0000
@@ -1684,9 +1684,6 @@
 ToFile( "CC",                $CC,                "e" );
 ToFile( "CXX",               $CXX,               "e" );
 ToFile( "USE_SYSTEM_STL",    "@USE_SYSTEM_STL@", "e" );
-if ( $COM ne "MSC" ) {
-  ToFile( "LINK",			 $CC,				 "e" );
-}
 ToFile( "ENABLE_SYMBOLS",    "@ENABLE_SYMBOLS@",    "e" );
 ToFile( "ENABLE_CRASHDUMP",  "@ENABLE_CRASHDUMP@",  "e" );
 ToFile( "ENABLE_CUPS",       "@ENABLE_CUPS@",       "e" );
